consensus/polybft/polybft_config.go

```diff
@@ -93,6 +93,8 @@ type BridgeConfig struct {
 	RootERC1155PredicateAddr  types.Address `json:"erc1155PredicateAddress"`
 	CustomSupernetManagerAddr types.Address `json:"customSupernetManagerAddr"`
 	StakeManagerAddr          types.Address `json:"stakeManagerAddr"`
+	// only populated if stake-manager-deploy command is executed, and used for e2e tests
```

Comment:
Deploy doesn't know what token was provided to `StakeManager` when that contract got deployed, since we only provide the `StakeManger` address to it. That new command for deploying the `StakeManager` updates this field in genesis, but as I said in the code comment, that field is only used in `e2e` tests, so that we can easily find the address of the staking token, without the need to call `StakeManager` contract.